pxb-tag/PUBLISHING.md

```diff
@@ -0,0 +1,11 @@
+# Publishing Instructions
+
+To update the version number, edit the version in `package.json`.
+
+To publish a new package through NPM, run the following commands from the root folder:
+
+```
+npm publish --tag <alpha | beta>
```

Comment:
shouldn't this be `yarn publish:package` for consistency with our other repositories?

Reply:
I'll add a `publish` script to the package.json so that we can publish locally.  Good catch